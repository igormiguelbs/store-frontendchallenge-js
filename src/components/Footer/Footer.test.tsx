import renderer from "react-test-renderer";
import { Footer } from ".";

describe("/Footer", () => {
  it("should render component correctly", () => {
    const result = renderer.create(<Footer />);
    expect(result).toMatchSnapshot();
  });
});
