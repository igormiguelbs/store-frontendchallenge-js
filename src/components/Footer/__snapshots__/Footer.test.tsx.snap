// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Footer should render component correctly 1`] = `
<div
  className="sc-bcXHqe eoSokX"
>
  <p>
    MKS sistemas Â© Todos os direitos reservados
  </p>
</div>
`;
